var/spack/repos/builtin/packages/mono/package.py

```diff
@@ -0,0 +1,65 @@
+###############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Mono(AutotoolsPackage):
+    """Mono is a software platform designed to allow developers to easily
+       create cross platform applications. It is an open source
+       implementation of Microsoft's .NET Framework based on the ECMA
+       standards for C# and the Common Language Runtime.
+    """
+
+    homepage = "http://www.mono-project.com/"
+    url      = "https://download.mono-project.com/sources/mono/mono-5.0.1.1.tar.bz2"
+
+    # /usr/share/.mono/keypairs needs to exist or be able to be
+    # created, e.g. https://github.com/gentoo/dotnet/issues/6
+    variant('patch-folder-path', default=False,
+            description='Point SpecialFolder.CommonApplicationData folder '
+            'into Spack installation instead of /usr/share')
+
+    # Spack's openssl interacts badly with mono's vendored
+    # "boringssl", don't drag it in w/ cmake
+    depends_on('cmake~openssl', type=('build'))
```

Comment:
Yep.  Just when you thought (you'd seen it all | it was safe to come out of hiding):

```
hartzelg-UXG8WL:mono-5.0.1.1 hartzelg$ ls config* *in *ac
Makefile.in		config.h.in		config.rpath		configure		configure.ac
config.guess		config.h.in		config.sub		configure.ac		mono-uninstalled.pc.in
hartzelg-UXG8WL:mono-5.0.1.1 hartzelg$ find . | grep CMak | head
./external/boringssl/CMakeLists.txt
./external/boringssl/crypto/aes/CMakeLists.txt
./external/boringssl/crypto/asn1/CMakeLists.txt
./external/boringssl/crypto/base64/CMakeLists.txt
./external/boringssl/crypto/bio/CMakeLists.txt
./external/boringssl/crypto/bn/CMakeLists.txt
./external/boringssl/crypto/buf/CMakeLists.txt
./external/boringssl/crypto/bytestring/CMakeLists.txt
./external/boringssl/crypto/chacha/CMakeLists.txt
./external/boringssl/crypto/cipher/CMakeLists.txt
```